examples/sample-operator/cmd/manager/main.go

```diff
@@ -92,8 +94,14 @@ func main() {
 
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
-		Namespace:          namespace,
-		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+		Cache: cache.Options{
+			DefaultNamespaces: map[string]cache.Config{
+				namespace: {},
+			},
+		},
```

Comment:
why not nil?

Reply:
wdym? this is needed to keep the behavior the same as before (`Namespace: namespace`)

Reply:
Never mind you are right.